jedi.inc

```diff
@@ -1955,6 +1974,12 @@
   {$DEFINE HAS_UNITSCOPE}
   {$DEFINE HAS_UNIT_SYSTEM_UITYPES}
   {$DEFINE HAS_UNIT_VCL_THEMES}
+  {$DEFINE HAS_UNIT_SYSTEM_ZIP}
+  {$DEFINE HAS_UNIT_SYSTEM_ZLIB}
+  {$DEFINE HAS_TSTRINGS_ENCODING}
+  {$IFDEF CPU64}
+    {$DEFINE EXTENDED_IS_DOUBLE_ALIAS}
+  {$ENDIF}
```

Comment:
Sorry for this late comment, but reading it once again, I believe this should not be in a RTL related section as the fact that `extended` is an alias for `double` is a feature of the 64 bit compiler. And so I feel these lines would be better placed in the appropriate `COMPILER` section

Reply:
Done